1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,25 +1,46 @@
-function setAlarm() {}
-
-// DO NOT EDIT BELOW HERE
-
-var audio = new Audio("alarmsound.mp3");
-
-function setup() {
-  document.getElementById("set").addEventListener("click", () => {
-    setAlarm();
-  });
-
-  document.getElementById("stop").addEventListener("click", () => {
-    pauseAlarm();
-  });
-}
-
-function playAlarm() {
-  audio.play();
-}
-
-function pauseAlarm() {
-  audio.pause();
-}
-
-window.onload = setup;
+
+let countDown
+function setAlarm() {
+  const timeRemaining = document.getElementById("timeRemaining");
+  document.querySelector("body").style.backgroundColor = "white";
+  //  get the input value
+  let alarmSet = document.getElementById("alarmSet").value;
+  // get the element to show ouput
+  countDown = setInterval(()=> {
+ alarmSet--;
+  timeRemaining.innerText = `Time Remaining: 00:${alarmSet}`;

```

Comment:
this is almost there Adiba, so well done. But notice the order of the lines here. When you input say 5, you'll notice the countdown on the screen actually starts at 4. This is because the `alarmSet` is being reduced by 1 before the` innerText` is actually updated, if you swap these lines around this should fix that issue :)